bundles/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/internal/LcnModuleDiscoveryService.java

```diff
@@ -153,8 +153,10 @@ protected void startScan() {
 
                             ThingUID bridgeUid = localBridgeHandler.getThing().getUID();
                             String serialNumber = matcher.group("sn");
-                            ThingUID thingUid = new ThingUID(LcnBindingConstants.THING_TYPE_MODULE, bridgeUid,
-                                    serialNumber);
+
+                            String thingID = String.format("S%dM%d", addr.getSegmentId(), addr.getModuleId());
```

Comment:
Add leading zeros to achieve sortable Thing IDs.
```suggestion
                            String thingID = String.format("S%03dM%03d", addr.getSegmentId(), addr.getModuleId());
```